### DISCUSSION TITLE
Excited for 0.20!!

### DISCUSSION THREAD

**Discussion by shanecowherd** [intitni/CopilotForXcode#297] [General]
I'm super excited for this next round of updates!

<img width="386" alt="Screenshot 2023-07-10 at 9 34 39 AM" src="https://github.com/intitni/CopilotForXcode/assets/442439/754d24d5-a7fa-4cde-a713-2b6586283a88">

**Comment by intitni** [comment]
Please don't be too excited, what you asked for is not there yet.
